cli/.gitignore

```diff
@@ -0,0 +1,8 @@
+dist
+build
+.tox
+env
+dcos_spark.egg-info
+tests/integration/__pycache__
+dcos_spark/data/spark**
+**.pyc
```

Comment:
Oh, does git honor gitignore files in subdirectories?  If so, then this is fine.